dev/io.openliberty.org.apache.myfaces.4.1/src/META-INF/beans.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+    Copyright (c) 2022 IBM Corporation and others.
```

Comment:
Is this just copied and the reason for no new copyright header?

Reply:
Correct - same beans.xml as with the other impl bundles.